src/BloomBrowserUI/bookEdit/js/textOverPicture.ts

```diff
@@ -31,16 +40,237 @@ class TextOverPictureManager {
         });
     }
 
+    public getIsCalloutEditingOn(): boolean {
+        return this.isCalloutEditingOn;
+    }
+
+    public turnOnBubbleEditing(): void {
+        if (this.isCalloutEditingOn === true) {
+            return; // Already on. No work needs to be done
+        }
+        this.isCalloutEditingOn = true;
+
+        Array.from(
+            document.getElementsByClassName("bloom-imageContainer")
+        ).forEach(e => e.classList.add("bloom-hideImageButtons"));
+        // todo: select one of them...make sure this doesn't conflict with any other strategy
+        // for selecting one we just added.
+        // todo: do this for the selected element, not just the first.
```

Comment:
These are slightly out of date. We ARE selecting one of them (the first). But still need to make sure that we select the right one (e.g., a newly added one) and that comical is turned on for the right parent (in case more than one image on page).